ui/components/ui/text-field/text-field.component.js

```diff
@@ -7,43 +7,46 @@ const inputLabelBase = {
   transform: 'none',
   transition: 'none',
   position: 'initial',
-  color: '#5b5b5b',
+  color: 'var(--color-text-default)',
 };
 
 const styles = {
   materialLabel: {
     '&$materialFocused': {
-      color: '#aeaeae',
+      color: 'var(--color-text-alternative)',
     },
     '&$materialError': {
-      color: '#aeaeae',
+      color: 'var(--color-text-alternative)',
     },
     fontWeight: '400',
-    color: '#aeaeae',
+    color: 'var(--color-text-alternative)',
   },
   materialFocused: {},
   materialUnderline: {
+    '&:before': {
+      borderBottom: '1px solid var(--color-text-default) !important', // Visible bottom border
```

Comment:
I added this `important` because of a rogue `:hover` effect which turns the border black with a horrifyingly hideous CSS selector.